### ISSUE TITLE
Question on monitoring.interceptor

### ISSUE THREAD

**Issue by Tenseiga** [confluentinc/confluent-kafka-dotnet#660]
Description
===========
Hi,

Is the Confluent Monitoring Interceptor is compatible with dotnet core?
I tried with both dotnet core and .net 4.5 as well. both giving LoadLibrary failed exception.
I have tried providing the absolute path as well

Thanks,

How to reproduce
================

Checklist
=========
Please provide the following information:

 - [ ] Confluent.Kafka nuget version: 1.0 exp 13
 - [ ] Apache Kafka version: 2.0
 - [ ] Client configuration:
 - [ ] Operating system: win 10
 - [ ] Provide logs (with "debug" : "..." as necessary in configuration)
 - [ ] Provide broker log excerpts
 - [ ] Critical issue

**Comment by mhowlett** [comment]
yes, it is. however, there's an unresolved issue whereby the monitoring interceptor can reference the wrong librdkafka version causing `Library.Load` to fail if additional versions are installed on your system. i'm guessing your problem is probably this. perhaps make sure there are no librdkafka versions on your system except the one in the librdkafka.redist package referenced by your version of `Confluent.Kafka`.

**Comment by jmathai-confluent** [comment]
Please reopen if this issue persists with more recent versions.
